src/simulation/dynamics/spinningBodies/spinningBodiesTwoDOF/spinningBodyTwoDOFStateEffector.h

```diff
@@ -51,12 +51,12 @@ class SpinningBodyTwoDOFStateEffector: public StateEffector, public SysModel {
     std::string nameOfTheta1DotState;   //!< -- identifier for the thetaDot1 state data container
     std::string nameOfTheta2State;      //!< -- identifier for the theta2 state data container
     std::string nameOfTheta2DotState;   //!< -- identifier for the thetaDot2 state data container
-    Eigen::Vector3d r_S1B_B;            //!< [m] vector pointing from body frame B origin to lower spinning frame S1 origin in B frame components
-    Eigen::Vector3d r_S2S1_S1;          //!< [m] vector pointing from lower spinning frame S1 origin to upper spinning frame S2 origin in S1 frame components
-    Eigen::Vector3d r_Sc1S1_S1;         //!< [m] vector pointing from lower spinning frame S1 origin to point Sc1 (center of mass of the lower spinner) in S1 frame components
-    Eigen::Vector3d r_Sc2S2_S2;         //!< [m] vector pointing from upper spinning frame S2 origin to point Sc2 (center of mass of the upper spinner) in S2 frame components
-    Eigen::Vector3d s1Hat_S1;           //!< -- first spinning axis in S1 frame components.
-    Eigen::Vector3d s2Hat_S2;           //!< -- second spinning axis in S2 frame components.
+    Eigen::Vector3d r_S1B_B{0.0,0.0,0.0};            //!< [m] vector pointing from body frame B origin to lower spinning frame S1 origin in B frame components
+    Eigen::Vector3d r_S2S1_S1{0.0,0.0,0.0};          //!< [m] vector pointing from lower spinning frame S1 origin to upper spinning frame S2 origin in S1 frame components
+    Eigen::Vector3d r_Sc1S1_S1{0.0,0.0,0.0};         //!< [m] vector pointing from lower spinning frame S1 origin to point Sc1 (center of mass of the lower spinner) in S1 frame components
+    Eigen::Vector3d r_Sc2S2_S2{0.0,0.0,0.0};         //!< [m] vector pointing from upper spinning frame S2 origin to point Sc2 (center of mass of the upper spinner) in S2 frame components
+    Eigen::Vector3d s1Hat_S1{0.0,0.0,0.0};           //!< -- first spinning axis in S1 frame components.
```

Comment:
Don't we want to initialize the unit vectors to ``[1,0,0]`` or something?

Reply:
Good point. Change made!